test/hotspot/jtreg/gc/shenandoah/oom/TestThreadFailure.java

```diff
@@ -78,6 +78,7 @@ public static void main(String[] args) throws Exception {
         {
             ProcessBuilder pb = ProcessTools.createLimitedTestJavaProcessBuilder(
                     "-Xmx32m",
+                    // "-XX:+UnlockExperimentalVMOptions", "-XX:ShenandoahNoProgressThreshold=12", "-XX:+ShowMessageBoxOnError",
```

Comment:
Ignore this. Will be removed; debugging detritus left in place until we believe we are ready.

Reply:
Removed in latest. Resolving.